lib/main.coffee

```diff
@@ -137,9 +152,14 @@ module.exports =
           return messages.map (message) ->
             startPoint = [message.line - 1, message.column - 1]
             endPoint = [message.line - 1, message.column]
-            return {
+            ret = {
               type: message.type
-              text: message.message
               filePath,
               range: [startPoint, endPoint]
             }
+            if showSource
```

Comment:
Either make the `->` starting this function a `=>`, or move the copy of the variable into the `lint()` function.

If you have the copy where it is currently and the user changes the setting after the initial call it won't take effect as the copy will have already been made.